### PR TITLE
br: record size in backupmeta

### LINKED ISSUE #48745: br: add size attribute to backup metadata
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)

Turn on backup_v2 at cloud, and run backup.

### 2. What did you expect to see? (Required)

The backup size attribute is shown correctly.

### 3. What did you see instead (Required)

The size is wrong.

### PR DESCRIPTION
**Pull request by Leavrth**
### What problem does this PR solve?

Issue Number: close #48745

Problem Summary:
record the backup size into the backupmeta so that other components can easily get the backup size.

### What is changed and how it works?
calculate the backup size and record it into backupmeta
### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)

after decode backupmeta, shown:
`"backup_size":238405687`
and the Size shown in the final backup success output is 
`[Size=237052687]`
the dfference between `backup_size` and `Size` is the size of the backupmeta and metafiles.

- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  >   

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -53,7 +53,7 @@ require (
 	github.com/google/btree v1.1.2
 	github.com/google/pprof v0.0.0-20211122183932-1daafda22083
 	github.com/google/skylark v0.0.0-20181101142754-a5f7082aabed
-	github.com/google/uuid v1.3.1
+	github.com/google/uuid v1.4.0
```
Comment (BornChanger) [inline go.mod:56]: Why upgrade libraries?

**Review by BornChanger — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (Leavrth) [reply go.mod:56]: The imported package by kvproto imports uuid v1.4.0

**Review by 3pointer — APPROVED** [review]

**Review by YuJuncen — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (Leavrth) [reply go.mod]: It seems `go get -u github.com/pingcap/kvproto` will also upgrade its deps. I will uses `go get github.com/pingcap/kvproto` instead.

**Pull request merged**